CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Updated SDK from 1.8 to to 1.13 ([polkadot-fellows/runtimes#322](https://github.com/polkadot-fellows/runtimes/pull/322)).
```

Comment:
There are a lot of user-facing changes covered by this one line. Do we expect the users to go to polkadot-sdk and go through the release notes of 1.8, 1.9, .. , 1.13 ?

I think we should list them here, at least the `Runtime User` ones. Either scrape them all (but it's going to be verbose and _a lot_ of information (some irrelevant like Rococo/Westend runtime changes), or manually pick important ones.

Reply:
Started on a proper changelog now here: https://github.com/polkadot-fellows/runtimes/pull/322/commits/b90443e4bf76b3653f41781a0a23bb15553bd28e